src/NodaTime/YearMonth.cs

```diff
@@ -319,11 +320,16 @@ int IComparable.CompareTo(object obj)
         /// <summary>
         /// Formats the value of the current instance using the specified pattern.
         /// </summary>
+        /// <remarks>
+        /// Unlike most <see cref="IFormattable"/> implementations, a <paramref name="patternText"/> of null with
+        /// the current thread's culture does not yield the same result as the parameterless <see cref="ToString()"/>
+        /// overload, for backward-compatibility reasons. (It uses the ISO format, which is culture-insensitive.)
+        /// </remarks>
         /// <returns>
         /// A <see cref="System.String" /> containing the value of the current instance in the specified format.
         /// </returns>
         /// <param name="patternText">The <see cref="System.String" /> specifying the pattern to use,
-        /// or null to use the default format pattern ("D").
```

Comment:
Just so I understand: I'm assuming this was a typo before; the default was actually `g` anyway?

Reply:
Correct. I think it was a copy/paste error.